src/main/java/org/cactoos/collection/Shuffled.java

```diff
@@ -52,16 +54,16 @@ public Shuffled(final T... src) {
      */
     public Shuffled(final Iterable<T> src) {
         super(
-            new Sticky<>(
-                new CollectionOf<>(
+            new Unchecked<>(
+                new org.cactoos.scalar.Sticky<>(
```

Comment:
@fanifieiev if I'm right, this replacement is also out of scope (`CollectionOf should be deleted`). Maybe it's better to leave a todo?

Reply:
@iakunin What will put instead of deleted `CollectionOf`?

Reply:
@fanifieiev please see https://github.com/yegor256/cactoos/pull/1256#discussion_r356656070

Reply:
@fanifieiev now (after https://github.com/yegor256/cactoos/pull/1256#discussion_r356657444) this replacement if fine for me, thank you.